src/lib/user-thumbnail.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @user-thumbnail
+ * Utility functions to return thumnail-related strings
+ */
+
+/**
+ * Generate a thumbnail url for a particular userid, with width and height.
+ * @param {string} userId userId for the user whose thumbnail we want
+ * @param {number} width desired thumbnail width; defaults to 32
+ * @param {number} height desired thumbnail height; defaults to width.
+ * @returns {string} thumbnail url string
+ */
+const thumbnailUrl = (userId, width, height) => (
+    `/get_image/user/${userId}_${width ? width : 32}x${height ? height : (width ? width : 32)}.png`
```

Comment:
This is _probably_ a temporary measure, but we should set it up as if it's permanent in case it doesn't end up temporary. If this goes live, we need these images to be cached, so this should use a hostname. In production we're using `cdn2.scratch.mit.edu` for `get_image` URLs. On Fastly we refer to this as "static", so maybe `process.env.STATIC_HOST`? Follow the pattern of `API_HOST` etc., to make sure it's set correctly in Travis as well. In development, we can set it to `""` and you'll get whatever you were seeing here without any host.

Reply:
OK, take a look